charts/openstack-cloud-controller-manager/values.yaml

```diff
@@ -10,6 +10,7 @@ commonAnnotations: {}
 #   "helm.sh/hook-delete-policy": before-hook-creation
 
 # Image repository name and tag
+imagePullSecrets: []
```

Comment:
This should probably be on the line above, to ensure that the comment `# Image repository name and tag` remains in the right place.

Furthermore, it would be great to add a similar comment that details how to use `imagePullSecrets` with an example.

Reply:
This was fixed, please re-check.